### DISCUSSION TITLE
how remove a monitor interface

### DISCUSSION THREAD

**Discussion by 011248163264** [Security-Onion-Solutions/securityonion#8220] [Unsupported Versions] [answered]
Hi, i can add a new monitor with sudo so-monitor-add but how can i remove a interface ?
I have 2 interfaces.

**Comment by mnasec** [comment, marked as answer]
All interfaces are bound as a slave interface to the bond0 interface. You may edit the config of the interface you want to remove from the bond0 interface.
Just look into the directory /etc/sysconfig/network-scripts, choose the right ifcfg-bond0-slave-xxx config file and set SLAVE=no and ONBOOT=no.

After a reboot his interface will not participate and the bond0 bridge anymore.

There is also the command ifenslave available: ifenslave -- Attach and detach slave network devices to a bonding device.

To see which interface participates in the bond0 interface, just look at
`cat /proc/net/bonding/bond0`

**Reply by 011248163264** [reply]
thanks
